resources/asciidoctor/spec/copy_images_spec.rb

```diff
@@ -532,8 +541,11 @@ def copy_attributes(copied)
       %w[coming note],
       %w[deprecated warning],
   ].each do |(name, admonition)|
-    it "copies images for the block formatted #{name} change admonition when requested" do
+    it "copies images for the block formatted #{name} change admonition" \
+        " when requested" do
       copied = []
+      relative_path = 'images/icons'
```

Comment:
I expect to remove this repetition in a follow up change where I switch this to more standard rspec.